File: Portifolio/Web/intranetsite/src/Gerenciamento/Empresa/Model/EmpresaDTO.ts
export class EmpresaDTO {
    CNPJ: string;
    RazaoSocial: string;
    NomeFantasia: string;

    

    constructor(CNPJ: string, RazaoSocial: string, NomeFantasia: string) {
        this.CNPJ = CNPJ;
        this.RazaoSocial = RazaoSocial
        this.NomeFantasia = NomeFantasia
    }   
}

File: portifolio/Portifolio/Portifolio.Dominio/DTOs/Empresa/EmpresaDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Portifolio.Dominio.DTOs.Empresa
{
    public class EmpresaDTO
    {
        public int IdEmpresa { get; set; }
        public string Cnpj { get; set; }
        public string RazaoSocial { get; set; }
        public string NomeFantasia { get; set; }
    }
}
